layouts/default.vue

```diff
@@ -74,11 +84,38 @@ export default {
                     href:
                         'https://fonts.googleapis.com/css2?family=Noto+Serif+TC:wght@400;500;600;700&family=Source+Sans+Pro:wght@400;600;700&display=swap',
                 },
-
                 {
                     rel: 'canonical',
                     href: 'https://tw.pycon.org' + this.$route.path,
                 },
+                {
+                    rel: 'alternate',
+                    hreflang: 'x-default',
+                    href: 'https://tw.pycon.org/2021/en-us',
+                },
+                {
+                    rel: 'alternate',
+                    hreflang: 'en-us',
+                    href: 'https://tw.pycon.org/2021/en-us',
+                },
+                {
+                    rel: 'alternate',
+                    hreflang: 'zh-hant-tw',
+                    href: 'https://tw.pycon.org/2021/zh-hant',
+                },
+            ],
+            script: [
```

Comment:
just for the record, these sites could be useful
- https://search.google.com/test/rich-results
- https://schema.org/Organization